bruno/collections/Interledger/environments/Local Playground.bru

```diff
@@ -0,0 +1,16 @@
+vars {
+  host3000: cloud-nine-wallet-backend
+  host4000: happy-life-bank-backend
+  clientWalletAddress: http://localhost:4000/accounts/pfry
+  clientKeyId: keyid-97a3a431-8ee1-48fc-ac85-70e2f5eba8e5
+  senderWalletAddress: http://localhost:3000/accounts/gfranklin
+  receiverWalletAddress: http://localhost:4000/accounts/asmith
+  RafikiGraphqlHost: http://localhost:3001
+  RafikiAuthGraphqlHost: http://localhost:3003
+  PeerGraphqlHost: http://localhost:4001
+  PeerAuthGraphqlHost: http://localhost:4003
+  signatureUrl: https://kxu5d4mr4blcthphxomjlc4xk40rvdsx.lambda-url.eu-central-1.on.aws/
+}
+vars:secret [
+  clientPrivateKey
```

Comment:
is the consensus in the end to keep this secret, even on local playground?

Reply:
If so I think we still need to add some instruction on where to get it. I'm still confused about that actually. I pulled the one from postman which worked, and I thought maybe I could generate a key and copy the base64 version from `rafiki.money` but that doesnt work. I suppose even if we hardcoded one that works (which I'm still a proponent of to reduce setup) we should have some documentation of where this comes from.